fire/inspectutils.py

```diff
@@ -24,6 +24,20 @@
 import six
 
 
+class _FullArgSpec(object):
```

Comment:
This is used outside of this module, albeit never reference by name outside of this module. So, I think we should drop the underscore. What do you think?